test/integration/policy_selector_test.go

```diff
@@ -0,0 +1,230 @@
+// Copyright (c) 2022 Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+
+package integration
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	policiesv1 "open-cluster-management.io/governance-policy-propagator/api/v1"
+
+	"github.com/stolostron/governance-policy-framework/test/common"
+)
+
+var _ = Describe(
+	"GRC: [P1][Sev1][policy-grc] Test objectSelector and context variables",
+	Ordered,
+	func() {
+		const (
+			policyName         = "policy-selector"
+			policyYAML         = "../resources/policy_selector/policy-selector.yaml"
+			configMapName      = "selector-config"
+			unrelatedConfigMap = "other-config"
+			configNamespace    = "policy-selector-test"
+			selectorKey        = "policy-selector-e2e"
+		)
+
+		configMapNames := []string{}
+		// Populate the ConfigMap names used in the tests (selector-config1, selector-config2, ...)
+		for i := 1; i <= 2; i++ {
```

Comment:
Will we ever have more than 2? It's not a big deal, but I feel like it'd be simpler to just pre-populate the list.

Reply:
Probably not--it's a fair point. I'll pre-populate instead. 😄